moped-database/migrations/1668201706589_create_phase_key/up.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE moped_phases ADD COLUMN phase_key text UNIQUE;
+COMMENT ON COLUMN "public"."moped_phases"."phase_key" IS E'Unique machine-readable phase name';
+UPDATE moped_phases set phase_key ='unknown' where phase_name = 'Unknown';
```

Comment:
Yeah, we still have this "Unknown" phase with ID `0`. Will address that in a subsequent PR.